CHANGELOG.md

```diff
@@ -6,7 +6,7 @@
 
 * The Mapbox Navigation SDK for iOS license has changed from the ISC License to the Mapbox Terms of Service. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808))
 * You can now install MapboxNavigation using Swift Package Manager, but you can no longer install it using Carthage. If you previously installed MapboxNavigation using Carthage, use Swift Package Manager instead. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808))
-* MapboxNavigation now depends on [MapboxMaps v10.0.0-beta.16](https://github.com/mapbox/mapbox-maps-ios/), MapboxNavigationNative v47.0.0, and MapboxCommon v10.0.2. ([#2884](https://github.com/mapbox/mapbox-navigation-ios/pull/2884))
+* MapboxNavigation now depends on [MapboxMaps v10.0.0-beta.18](https://github.com/mapbox/mapbox-maps-ios/), MapboxNavigationNative v48.0.4, and MapboxCommon v11.0.2. ([#2884](https://github.com/mapbox/mapbox-navigation-ios/pull/2930))
```

Comment:
Also add this changelog entry under “Location tracking”:

* Added the `RouteController.historyFileURL` property and `RouteController.writeHistory(completionHandler:)` method for recording details about a trip for debugging purposes. ([#2930](https://github.com/mapbox/mapbox-navigation-ios/pull/2930))